client/src/app/site/motions/services/motion-csv-export.service.ts

```diff
@@ -61,4 +61,17 @@ export class MotionCsvExportService {
             this.translate.instant('Call list') + '.csv'
         );
     }
+
+    public exportDummyMotion(): void {
+        const dummyData = ['Identifier', 'Title', 'Text', 'Reason', 'Submitters',
```

Comment:
Make from this first map statement a variable called `headerrow`.
then make a list like this: `rows = [headerrow, 'row 1', 'row 2', ...].
Return a `rows.join('\n')`